drivers/sensor/lis2dh/lis2dh_trigger.c

```diff
@@ -385,11 +381,16 @@ int lis2dh_init_interrupt(const struct device *dev)
 	 */
 
 	/* setup data ready gpio interrupt */
-	lis2dh->gpio_int1 = device_get_binding(cfg->irq1_dev_name);
-	if (lis2dh->gpio_int1 == NULL) {
-		LOG_INF("Cannot get pointer to irq1_dev_name");
+	if (!device_is_ready(cfg->gpio_drdy.port)) {
+		/* API may return false even when ptr is NULL */
+		if (cfg->gpio_drdy.port != NULL) {
+			LOG_ERR("Cannot get pointer to gpio_drdy device");
```

Comment:
One last thing, I guess in this case you do have a pointer, it's just not ready. Usually we do something like this then:

```
LOG_ERR("device %s is not ready", cfg->gpio_drdy.port->name);
```

Same for the other device.

Reply:
sounds good